medium/unique-paths/problem-statement0.md

```diff
@@ -0,0 +1,6 @@
+https://leetcode.com/problems/lru-cache
+
+Design and implement a data structure for Least Recently Used (LRU) cache. It should support the following operations: `get` and `set`.
```

Comment:
Surprisingly medium/unique-paths has two problem statements. Not sure how. Will submit a new request.